### DISCUSSION TITLE
Github Verification system is not working for Microsoft Azure

### DISCUSSION THREAD

**Discussion by Abdul-Rehman4** [community/community#140395] [GitHub Education]
### Hi everyone,

I'm facing a problem related Microsoft Azure, When I try to signup It says: We couldn't verify that you are a student , I talked to their support and they said Github has its own verification system & it is working well for other Services but not Only for Azure, I tried using my University mail but still declined and I can't even verify from github while signing up, Please resolve my Issue.
![Capture](https://github.com/user-attachments/assets/e8845dc0-4f44-4f9a-8b4e-d73474cd16f5)

**Comment by davevad93** [comment]
Hi @Abdul-Rehman4 ,

This is a known issue, #131336, #135739, to solve it you can do the following:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

The users in the discussions I provided above solved it by doing that.

**Reply by davevad93** [reply]
In your case, if you have contacted Azure support and are still unable to access the offer, you should contact GitHub support and select the dropdown I described above.

**Reply by Abdul-Rehman4** [reply]
Thankyou for Your Help @davevad93, Let me generate a ticket on github support

**Reply by davevad93** [reply]
You're welcome @Abdul-Rehman4 👍
